src/containers/CollectionPage.js

```diff
@@ -37,10 +39,17 @@ class CollectionPage extends React.Component {
   };
 
   render() {
-    const { collections, collection, publicFolder, page, entries } = this.props;
+    const { collections, collection, publicFolder, page, entries, isFetching } = this.props;
     if (collections == null) {
       return <h1>No collections defined in your config.yml</h1>;
     }
+
+    const noEntriesContent = isFetching
```

Comment:
Are you saying you'd like the `entries` and `isFetching ternaries to be nested or did you mean something like this?

```js
    const fetchingEntriesContent = (<Loader active>
        {['Loading Entries', 'Caching Entries', 'This might take several minutes']}
    </Loader>);

    const noEntriesContent = <div className={styles.noEntries}>No Entries</div>;

    const fallbackContent = isFetching ? fetchingEntriesContent : noEntriesContent;

    return (<div>
      {entries ?
        <EntryListing
          collections={collection}
          entries={entries}
          publicFolder={publicFolder}
          page={page}
          onPaginate={this.handleLoadMore}
        >
          {collection.get('label')}
        </EntryListing>
        :
        fallbackContent
      }
    </div>);
```